README.md

```diff
@@ -65,19 +65,19 @@ This will start the local Hugo server on port 1313. Open up your browser to <htt
 
 The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
 
-To update the reference pages for a new Kubernetes release (replace v1.20 in the following examples with the release to update to):
+To update the reference pages for a new Kubernetes release follow these steps:
 
-1. Pull the `kubernetes-resources-reference` submodule:
```

Comment:
The submodule name seems to have changed -- it isn't `kubernetes-resources-reference` anymore.

Reply:
This step repeats the instructions found on lines 32-35. 

```bash
# pull in the Docsy submodule
git submodule update --init --recursive --depth 1
```

Reply:
Right. I'll address that separately. The point of this change was to fix the misnamed submodule.